test/integration/hook-err.spec.js

```diff
@@ -267,6 +267,52 @@ describe("hook error handling", function () {
     });
   });
 
+  describe("--fail-hook-affected-tests", function () {
+    describe("before hook error", function () {
```

Comment:
I know this will dupe the beginning of each suite name, but really want to clarify these meta-names. We are not at a position prior to a hook error, we are referring to the hook named `before`.

```suggestion
    describe("error in `before` hook", function () {
```